### PR TITLE
fix AmbiguousTimeError at paypal.standard.forms.PayPalDateTimeField.strptime

### PR DESCRIPTION
**Pull request by yuriiz**

### CODE REVIEW

File: paypal/standard/ipn/tests/test_ipn.py [hunk 1]
```diff
@@ -360,8 +360,8 @@ def test_paypal_date_format(self):
             "subscr_effective": b("23:04:06 Jan 02, 2009 PST"),
             "auction_closing_date": b("23:04:06 Jan 02, 2009 PST"),
             "retry_at": b("23:04:06 Jan 02, 2009 PST"),
-            "case_creation_date": b("23:04:06 Jan 02, 2009 PST"),
-            "time_created": b("23:04:06 Jan 02, 2009 PST"),
+            "case_creation_date": b("01:13:05 Nov 01, 2015 PST"),
+            "time_created": b("01:13:05 Nov 01, 2015 PDT"),
```
Comment (spookylukey) [inline paypal/standard/ipn/tests/test_ipn.py]: Could you add a comment here to say this is deliberately in a DST cross-over period to test a corner case?

Comment (yuriiz) [inline paypal/standard/ipn/tests/test_ipn.py]: Added.
